sdk/appservice/azure-mgmt-web/HISTORY.rst

```diff
@@ -3,6 +3,13 @@
 Release History
 ===============
 
+0.43.1 (2019-10-17)
++++++++++++++++++++
+
+**General**
+
+- Remove obsolete autogenerated file which brings bug for multiapi
```

Comment:
Perhaps we should change it to something more generic
- Fixed incorrectly generated multi-api package structure